Jenkinsfile

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright (c) 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+edgeXBuildGoApp (
+    project: 'device-sdk-go',
+    goVersion: '1.11',
+    dockerImageName: 'docker-device-sdk-simple',
+    dockerFilePath: 'example/cmd/device-simple/Dockerfile',
+    dockerBuildContext: 'example/cmd/device-simple'
```

Comment:
We should add `pushImage: false` for the device-sdk-go we build the image but don't push it. Since we don't release a docker image for the Go Device SDK.

Reply:
Sounds good. I will update.